packages/mobile/src/account/Settings.tsx

```diff
@@ -258,9 +263,36 @@ export class Account extends React.Component<Props, State> {
     ValoraAnalytics.track(SettingsEvents.tos_view)
   }
 
+  onRemoveAccountPress = () => {
+    this.setState({ showAccountKeyModal: true })
+  }
```

Comment:
The way we have to define these handlers for each dialog makes me think we should just have the dialog manage its own visibility. Not something you have to fix, just thinking out loud.

Reply:
I can see how the dialogs could handle disappearing automatically when a button is pressed, but we would still need to tell the dialog somehow when to appear, right? I can't think of a good way of doing that without a handler like this.